<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogMotion</class>
 <widget class="QDialog" name="DialogMotion">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>652</width>
    <height>497</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {background-color: rgb(50, 50, 50);}</string>
  </property>
  <widget class="QPushButton" name="pushButton_left">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>190</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color: qlineargradient(spread:pad, x1:0.534, y1:0.289773, x2:0.534, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255)); color: white;}</string>
   </property>
   <property name="text">
    <string>&lt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_right">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>190</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color: qlineargradient(spread:pad, x1:0.534, y1:0.289773, x2:0.534, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255)); color: white;}</string>
   </property>
   <property name="text">
    <string>&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_up">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>140</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color: qlineargradient(spread:pad, x1:0.534, y1:0.289773, x2:0.534, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255)); color: white;}</string>
   </property>
   <property name="text">
    <string>up</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_down">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>230</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color: qlineargradient(spread:pad, x1:0.534, y1:0.289773, x2:0.534, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255)); color: white;}</string>
   </property>
   <property name="text">
    <string>down</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>190</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color: qlineargradient(spread:pad, x1:0.534, y1:0.289773, x2:0.534, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255)); color: white;}</string>
   </property>
   <property name="text">
    <string>HOME</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_cancel">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>440</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color: qlineargradient(spread:pad, x1:0.534, y1:0.289773, x2:0.534, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255)); color: white;}</string>
   </property>
   <property name="text">
    <string>Закрыть</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_mm">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>70</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <double>170.000000000000000</double>
   </property>
   <property name="value">
    <double>5.000000000000000</double>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_stop">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>300</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color: qlineargradient(spread:pad, x1:0.534, y1:0.289773, x2:0.534, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255)); color: white;}</string>
   </property>
   <property name="text">
    <string>STOP</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_pos">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>180</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color: qlineargradient(spread:pad, x1:0.534, y1:0.289773, x2:0.534, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255)); color: white;}</string>
   </property>
   <property name="text">
    <string>Pos</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_unlook">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>270</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color: qlineargradient(spread:pad, x1:0.534, y1:0.289773, x2:0.534, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255)); color: white;}</string>
   </property>
   <property name="text">
    <string>Unlook</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>370</y>
     <width>451</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>340</y>
     <width>441</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLineEdit" name="lineEdit"/>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_OK">
      <property name="styleSheet">
       <string notr="true">QPushButton{
background-color: qlineargradient(spread:pad, x1:0.534, y1:0.289773, x2:0.534, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255)); color: white;}</string>
      </property>
      <property name="text">
       <string>OK</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>91</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>17</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>X</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_x">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>17</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>0.00</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>100</y>
     <width>91</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <pointsize>17</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Y</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_y">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>17</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>0.00</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
